docs/en_US/Compression/Quantizer.rst

```diff
@@ -102,6 +102,43 @@ The quantizer will automatically detect Conv-BN patterns and simulate batch norm
 graph. Note that when the quantization aware training process is finished, the folded weight/bias would be restored after calling
 `quantizer.export_model`.
 
+Quantization dtype and scheme customization
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+QAT quantizer support dtype (int or uint) and scheme (per-tensor or per-channel and symmetric or affine) customization.
```

Comment:
Suggest adding why we add dtype and scheme here. For instance, different inference frameworks support various and specific quantization strategies, nni aims to support different inference frameworks targeting popular devices.

Reply:
done